Lib/fontTools/varLib/__init__.py

```diff
@@ -463,46 +463,134 @@ def _merge_TTHinting(font, masterModel, master_ttfs, tolerance=0.5):
 		var = TupleVariation(support, delta)
 		cvar.variations.append(var)
 
+hvar_fields = { 'table_tag': 'HVAR',
+	'metrics_tag': 'hmtx',
+	'sb1': 'LsbMap',
+	'sb2': 'RsbMap',
+	'mapping_name': 'AdvWidthMap',
+	}
+vvar_fields = { 'table_tag': 'VVAR',
+	'metrics_tag': 'vmtx',
+	'bearing1': 'TsbMap',
+	'bearing2': 'BsbMap',
+	'mapping': 'AdvHeightMap',
+	}
```

Comment:
These two dicts hvar_fields and vvar_fields are unused and replaced by the namedtuples with the same name below, should be removed